src/root_span_builder.rs

```diff
@@ -56,11 +56,10 @@ impl RootSpanBuilder for DefaultRootSpanBuilder {
 
 fn handle_error(span: Span, error: &actix_web::Error) {
     let response_error = error.as_response_error();
-    span.record(
-        "exception.message",
-        &tracing::field::display(response_error),
-    );
-    span.record("exception.details", &tracing::field::debug(response_error));
+    let display = format!("{}", response_error);
```

Comment:
Let's add a comment here linking to the issue on the tracing repository explaining why we are doing this.